api/wasm/cpp/proxy_wasm_impl.h

```diff
@@ -631,9 +631,19 @@ inline google::protobuf::Struct Context::getResponseMetadataStruct(StringView na
   return getMetadataStruct(MetadataType::Response, name);
 }
 
-// Continue
+// Continue/Respond/Route
 inline void continueRequest() { proxy_continueRequest(); }
 inline void continueResponse() { proxy_continueResponse(); }
+inline void sendLocalResponse(uint32_t response_code, StringView response_code_details, StringView body,
+    const HeaderStringPairs& additional_response_headers, uint32_t grpc_status = GrpcStatus::InvalidCode) {
```

Comment:
there is an implicit int32_t to uint32_t. See my suggestion in EnumType

Reply:
Made the conversion explicit.